apps/onboarding/src/session/session.service.ts

```diff
@@ -1,15 +1,30 @@
 import { normalizeAddress, trimLeading0x } from '@celo/base'
-import { Injectable } from '@nestjs/common'
+import { Inject, Injectable } from '@nestjs/common'
+import { ConfigType } from '@nestjs/config'
 import { QueryDeepPartialEntity } from 'typeorm/query-builder/QueryPartialEntity'
 import { v4 as uuidv4 } from 'uuid'
-import { Session } from './session.entity'
+import { Session, SessionQuota } from './session.entity'
 import { SessionRepository } from './session.repository'
+import { quotaConfig } from '../config/quota.config'
+import { ApiError } from '../errors/api-error'
 
+class QuotaExceededError extends ApiError<'QuotaExceededError'> {
+  statusCode = 403
```

Comment:
Let's make it a [429](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429)